### PR TITLE
Reintroduce deprecations from Base

### PR DESCRIPTION
**Pull request by ararslan**
@tkelman Just to confirm, is this it in terms of the deprecations to include here?

### CODE REVIEW

**Comment by tkelman** [comment]
plan on including the file?

**Comment by ararslan** [comment]
GAH! OOPS. This is why we have reviews. 🙂 Thanks! Anything else?

File: src/deprecated.jl [hunk 1]
```diff
@@ -1,15 +1,39 @@
 # This file contains code that was formerly a part of Julia. License is MIT: http://julialang.org/license
 
+# For deprecating vectorized functions in favor of compact broadcast syntax
+macro dep_vectorize_1arg(S, f)
```
Comment (tkelman) [inline src/deprecated.jl]: aren't these in Compat?

Comment (ararslan) [inline src/deprecated.jl]: Ah, so they are! I had no idea. Thanks, I'll use that instead.

**Comment by ararslan** [comment]
Good to go?

**Comment by tkelman** [comment]
haven't compared to base in a huge amount of detail, but yeah probably

**Comment by ararslan** [comment]
The only difference from Base is removing the `Base.Math.` prefix for the internal functions, since those are now defined in the package.

**Pull request merged**